client/src/js/components/bhFundingSourceSelect/bhFundingSourceSelect.js

```diff
@@ -0,0 +1,55 @@
+angular.module('bhima.components')
+  .component('bhFundingSourceSelect', {
+    templateUrl : 'js/components/bhFundingSourceSelect/bhFundingSourceSelect.html',
+    controller  : FundingSourceSelectController,
+    transclude  : true,
+    bindings    : {
+      fundingSourceUuid : '<',
+      onSelectCallback  : '&',
+      required          : '<?',
+      label             : '@?',
+      enableAdd         : '<?',
```

Comment:
This is cool! 👍 

If you don't need to dynamically update this, I would recommend you make it a `@` binding instead of a `<` binding to remove a `$watch`-er on this component interface.